### DISCUSSION TITLE
How should I set this up (Indexing / Importing / Folders)

### DISCUSSION THREAD

**Discussion by ITCMD** [photoprism/photoprism#1316] [Q&A] [answered]
Whenever I take a photo, it is uploaded via FTP to a folder. This folder has a good 2GB of photos so far.

Ideally, I would like the photo to show up in Photoprism within a couple of minutes of the photo being uploaded.

Here are my concerns:

- Originally I figured that I should set it to be an import folder, however, if Photoprism moves the picture from that folder, my phone will think the file is missing and will upload it again. This cycle would repeat forever
- If I set it to copy the pictures instead of moving them, my library will take twice the amount of space
- If I set the folder to be indexed, from what I understand files will not show up until I index the folder.

Am I missing a workaround of some sort?

**Comment by lastzero** [comment, marked as answer]
It's easiest to use WebDAV instead of the FTP, see https://docs.photoprism.org/user-guide/sync/webdav/

If you must use FTP, then you may upload directly to originals instead of import. Importing is strictly optional. In addition, you may use import in copy instead of move mode so that the files remain in the folder.

Here's a config example that shows how to run background jobs like automated indexing:

https://dl.photoprism.org/docker/scheduler/

If you use WebDAV, indexing is automatically performed when files were added and updated.

**Comment by ITCMD** [comment]
Gotchya. What ports and other info would webdav use?

**Reply by lastzero** [reply]
see https://docs.photoprism.org/user-guide/sync/webdav/
